client/src/components/Home/index.tsx

```diff
@@ -314,13 +314,27 @@ export default class Home extends React.Component<any, IHomeState> {
     );
 
     // Dividing templates into categories
```

Comment:
Please add a small comment:
General - All dashboards without any categories
Features - Dashboards appearing at the top of the creation screen